src/main/java/huyang/TaskFactory.java

```diff
@@ -0,0 +1,68 @@
+package huyang;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+public class TaskFactory {
+    private static final String DATE_PATTERN = "yyyy-MM-dd HHmm";
```

Comment:
Would it be possible to add more date patterns or make the time optional?

Reply:
Yes, you can always change the date pattern by editing what the static final variable is. To make it optional, simply comment this out and replace DATE_PATTERN with whatever date pattern you want.